.jenkins/pytorch/macos-build.sh

```diff
@@ -25,6 +25,10 @@ export CMAKE_PREFIX_PATH=${PYTORCH_ENV_DIR}/miniconda3/
 export MACOSX_DEPLOYMENT_TARGET=10.9
 export CXX=clang++
 export CC=clang
+if which ccache > /dev/null; then
```

Comment:
Since our macos workers use brew to install ccache, you can instead add "/usr/local/opt/ccache/libexec/" to `$PATH` instead.